code/src/index.css

```diff
@@ -1,13 +1,341 @@
+	margin-right: 4px;
+}
+
+header {
+	height: 12vh;
+	top: 0;
+	z-index: 1;
+	background-color: #000;
+	width: 100%;
+	padding: 0.5rem;
+	display: flex;
+	justify-content: center;
+	flex-direction: column;
+}
+
+header h1 {
+	font-size: 30px;
+  background: linear-gradient(to right, #8ccda3 0%, #09b3e3 50%);
+  -webkit-background-clip: text;
+  -webkit-text-fill-color: transparent; 
+	font-family: Waltograph, sans-serif;
+	background-color: black;
+}
+
+header h2 {
+	font-size: 20px;
+}
+
+/* MOVIE LIST */
+.movie-wrapper {
+	display: flex;
+	flex-wrap: wrap;
+	scroll-margin-top: 10rem;
+	justify-content: center;
+}
+
+.movie {
+	position: relative;
+	width: 45%;
+	margin: 0.5rem;
+}
+
+.movie-img {
+	width: 100%;
+}
+
+.overlay {
+	display: none;
+}
+
+/* MOVE DETAIL PAGE */
+.movie-page {
+	display: flex;
+	min-height: 85vh;
+	min-width: 50vh;
+	padding: 2rem;
+	background-size: cover;
+	flex-direction: column;
+	justify-content: flex-end;
+	align-items: center;
+	position: relative;
+}
+
+.movie-page > img {
+	object-fit: fill;
+}
+
+.rating {
+	display: flex;
+	color: #000;
+	font-size: 20px;
+	font-weight: 500;
+	background: #fff;
+	padding: 0 5px;
+	min-height: 2rem;
+	min-width: 70px;
+	margin-left: 1rem;
+	max-height: 2rem;
+	align-items: center;
+	justify-content: center;
+}
+
+.movie-detail-info {
+	display: flex;
+	flex-direction: column;
+	color: #fff;
+	align-items: center;
+	width: 100%;
+	max-height: 90vh
+}
+
+.movie-poster {
+	border: 3px solid #fff;
+	width: 60%;
+	margin-bottom: 1rem;
+	justify-self: center;
+}
+
+.rated-title {
+	display: flex;
+	margin: 1rem 0;
+	align-self: flex-start;
 }
 
-code {
-  font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
-    monospace;
+.movie-text {
+	margin: 1rem 0 2rem 1rem;
 }
+
+.movie-info {
+	margin-left: 1.5rem;
+	position: relative;
+}
+
+/* PAGE NOT FOUND */
+#not-found-container {
+	display: flex;
+	flex-direction: column;
+	align-self: center !important;
+	height: 80vh;
+	width: 100vw;
+	justify-content: center !important;
+	text-align: center !important;
+	align-items: center;
+}
+
+.ghost {
+	margin: 2rem;
+}
+
+.not-found-container h1, h2 {
+	color: #fff;
+}
+
+/* FOOTER */
+footer {
+	display: flex;
+	width: 100%;
+	justify-content: center;
+	align-items: center;
+	gap: 2rem;
+	position: fixed;
+	bottom: 0;
+	left: 0;
+	background-color: #000;
+	padding: 0.5rem;
+}
+
+ footer p {
+	color: #adadad;
+ }
+
+.tmdb-logo {
+	height: 1.5rem;
+}
+
+.site-title {
+	font-family: Waltograph, sans-serif;
+}
+
+/* LOADING SCREEN */
+.loading{
+	display: flex;
+	flex-direction: column;
+	align-items: center;
+	gap: 2rem;
+	color: #fff;
+	margin: auto;
+  width: 100%;
+	height: 100%;
+}
+
+.loading p {
+	font-size: 28px;
+}
+
+.loading-img {
+  display:block;
+  width: 100px;
+  height: 100px;
+  animation:loading-cog 5s linear infinite;
+}
+
+@keyframes loading-cog
+{
+  0%
+  {
+    transform:rotate(0deg);
+  }
+  100%
+  {
+    transform:rotate(360deg);
+  }
+}
```

Comment:
Love that you made this as an animation, almost made us wish the page wouldn't load as fast. 😎